tools/bouncer_worker/src/queues/unityQueueHandler.js

```diff
@@ -21,9 +21,11 @@ const { generateAssetBundles, validateUnityConfigurations } = require('../tasks/
 const { ERRCODE_ARG_FILE_FAIL, ERRCODE_BUNDLE_GEN_FAIL } = require('../constants/errorCodes');
 const { UNITY_PROCESSING } = require('../constants/statuses');
 const logger = require('../lib/logger');
+const Elastic = require('../lib/elastic');
 
 const logLabel = { label: 'UNITYQ' };
 
+// eslint-disable-next-line max-len
```

Comment:
do you need this? since you didn't change the next line and it never complained about it before?